packages/mermaid/src/mermaidAPI.ts

```diff
@@ -186,13 +169,11 @@ export const cssImportantStyles = (
  * Create the user styles
  *
  * @param  config - configuration that has style and theme settings to use
- * @param graphType - used for checking if classDefs should be applied
  * @param  classDefs - the classDefs in the diagram text. Might be null if none were defined. Usually is the result of a call to getClasses(...)
  * @returns  the string with all the user styles
  */
 export const createCssStyles = (
   config: MermaidConfig,
-  graphType: string,
```

Comment:
FYI, to other reviewers, this change is **NOT** a breaking change, since the `createCssStyles` function is not exported publicly in the `mermaidAPI` object.

Although, this is a bit confusing, so maybe in a future PR we should move `createCssStyles` out of the `mermaidAPI.ts` file in a future PR?

@sidharthv96, do you think it makes sense to mark this function as [`@internal`](https://tsdoc.org/pages/tags/internal/) so that it doesn't appear on the https://mermaid.js.org/config/setup/modules/mermaidAPI.html page? (this can be done in another PR).

Reply:
Done. 
But it didn't change any docs automatically.

IMO, we should deprecate and mark mermaidAPI itself as internal.

Reply:
https://github.com/mermaid-js/mermaid/pull/4821

Reply:
> But it didn't change any docs automatically.

Huh, weird. I swear we've used `@alpha`/`@experimental` before, and it worked to not export any documentation to the website, but maybe I'm just imaging things.

:shrug: Oh well

> IMO, we should deprecate and mark mermaidAPI itself as internal.

I'm not sure if I full agree with this, since marking something as `@internal` when it can be used by users doesn't seem right to me, but `@deprecate`-ing some of the functions does make sense!